src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -2053,6 +2053,30 @@ public void PreviousSubmissionWithError()
             Assert.Throws<InvalidOperationException>(() => CreateSubmission("a + 1", previous: s0));
         }
 
+        [Fact]
+        public void CreateTupleTypeSymbol()
```

Comment:
@jcouv it's possible the change to add `CompilerTrait` hasn't yet sync'd into the tuple branch.  If it's not resolvable please add a work item to add it back later once the branches merge. 

>  But I don't much care about test counts as a measure.

It provides value in the following two ways:

1. Roslyn has ~65,000 tests layered and ordered in no standard way.  This makes it impossible for people, aka leads, to ask questions like "what all tests do we have against tuples, or what tests have tuples and dynamic"?  Having this data makes it possible to audit test sets to determine if there are gaps that need to be plugged.  
2. It's by no means "the" metric for quality but it is "a" metric.  It's useful to comparce similar features for test size to see if there is a gap worth looking into.  If Local functions have 50 tests and tuples 500 then we likely have an issue.